docs/schema.adoc

```diff
@@ -1,8 +1,17 @@
-[cols="1m,3"]
+[cols="1m,2a"]
 |===
-| CALL apoc.schema.existsIndex(indexLabel, propertyName) yield value | it returns true if an index exists for a label "indexLabel" and a property "propertyName"
-| CALL apoc.schema.existsConstraintForNode(indexLabel, propertyName, constraintType) yield value | it returns true if a constraint exists for a label "indexLabel" and a property "propertyName". The constraint can be a uniqueness constraint or an existence constraint so you need to pass the third parameter to specify what type of constraints you are looking for. Possible values are: UNIQUENESS or NODE_PROPERTY_EXISTENCE.
-| CALL apoc.schema.existsConstraintForRelationship(relationshipType, propertyName) yield value | it returns true if a constraint exists for a relationship type "relationshipType" and a property "propertyName".
+| CALL apoc.schema.nodes() yield name, label, properties, status, type
+| * name of the index/constraint,
```

Comment:
also as this is a "schema.adoc" you don't need to keep the tabular format, just format it so that the page looks good
in terms of examples + list of results columns + general syntax